src/features/donations/components/treeDonation/DirectGift.tsx

```diff
@@ -11,11 +13,16 @@ export default function DirectGift({
   directGift,
   setShowDirectGift,
 }: Props): ReactElement {
+  const { t } = useTranslation(['donate', 'common']);
   return (
     <div className={styles.giftContainer}>
       <div className={styles.textContainer}>
-        <div className={styles.giftTo}>Gift to {directGift.displayName}</div>
-        <div className={styles.selectProject}>Select a project</div>
+        <div className={styles.giftTo}>
+          {directGift.type === 'individual'
+            ? t('donate:giftToName', { name: directGift.displayName })
+            : t('donate:plantTreesWith', { name: directGift.displayName })}
```

Comment:
This is the text for supporting TPOs? But we do not show the support link - so can I test this?

Reply:
I will test this once it's merged with a educational profile I have created.